account.go

```diff
@@ -157,22 +157,22 @@ func (account *Account) fmtExecute(ctx context.Context, calls []types.Transactio
 	return &req, nil
 }
 
-func fmtExecuteCalldataStrings(nonce *big.Int, calls []types.Transaction) (calldataStrings []string) {
+func fmtExecuteCalldataStrings(nonce *big.Int, calls []types.Transaction) (calldataStrings []*types.Felt) {
```

Comment:
From a check, this function is (1) not exported because it starts with a lowercase `f` AND (2) only used once in the original code which should change because transaction will be Felts too. My guess is that it does not make sense anymore